02_bubble/brvishnu92/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBubble(t *testing.T) {
```

Comment:
You could do some table tests for both bubble and insertion sorting.

Reply:
Sure. added it :-) thanks